dept44-starter/src/test/java/se/sundsvall/dept44/util/DateUtilsTest.java

```diff
@@ -1,45 +1,63 @@
 package se.sundsvall.dept44.util;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
 import java.time.OffsetTime;
+import java.time.ZoneId;
 import java.time.ZonedDateTime;
 import java.time.chrono.ThaiBuddhistDate;
 import java.time.temporal.Temporal;
 import java.util.stream.Stream;
 
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.Arguments;
-import org.junit.jupiter.params.provider.MethodSource;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 class DateUtilsTest {
 
 	@Test
 	void toOffsetDateTimeWithLocalOffset() {
 
-		// Zulu time zone into time with local offset +1h.
-		assertThat(DateUtils.toOffsetDateTimeWithLocalOffset(OffsetDateTime.parse("2021-11-10T09:23:42.500Z"))).hasToString("2021-11-10T10:23:42.500+01:00");
+		// Create Offset
+		final var zoneOffsetWinter = OffsetDateTime.parse("2021-11-10T09:23:42.500Z").atZoneSameInstant(ZoneId.systemDefault()).getOffset();
+		final var zoneOffsetSummer = OffsetDateTime.parse("2021-06-10T09:23:42.500Z").atZoneSameInstant(ZoneId.systemDefault()).getOffset();
+
+		final var winterOffsetDateTimeWithoutZuluTimeZone = OffsetDateTime.parse("2021-11-10T09:23:42.500Z");
+		final var summerOffsetDateTimeWithoutZuluTimeZone = OffsetDateTime.parse("2021-06-10T09:23:42.500Z");
+		final var winterOffsetDateTimeWithPlusFour = OffsetDateTime.parse("2021-11-10T12:23:42.500+04:00");
+		final var winterOffsetDateTimeWithMinusOne = OffsetDateTime.parse("2021-11-10T12:23:42.500-01:00");
+		final var winterLocalDateTimeNoOffset = LocalDateTime.parse("2021-11-10T12:23:42");
+		final var summerLocalDateTimeNoOffset = LocalDateTime.parse("2021-06-10T12:23:42");
+
+		// Zulu time zone into time with local offset +1h in Sweden.
+		assertThat(DateUtils.toOffsetDateTimeWithLocalOffset(winterOffsetDateTimeWithoutZuluTimeZone))
+			.hasToString(winterOffsetDateTimeWithoutZuluTimeZone.atZoneSameInstant(zoneOffsetWinter).toString());
 
-		// Zulu time zone into time with local offset +2h, with DST.
-		assertThat(DateUtils.toOffsetDateTimeWithLocalOffset(OffsetDateTime.parse("2021-06-10T09:23:42.500Z"))).hasToString("2021-06-10T11:23:42.500+02:00");
+		// Zulu time zone into time with local offset +2h, with DST in Sweden.
+		assertThat(DateUtils.toOffsetDateTimeWithLocalOffset(summerOffsetDateTimeWithoutZuluTimeZone))
+			.hasToString(summerOffsetDateTimeWithoutZuluTimeZone.atZoneSameInstant(zoneOffsetSummer).toString());
```

Comment:
This test doesn't ensure that date transformation is "+02:00", (or it is at the best very unclear...)
(See the text above the test)

All changes in this class reduces readability very much. Why?

Reply:
Just because building in GitHub-environment did not work due to different timezone. In Sweden, as described, it results to local offset +2. 
Why the reduce of readability? Because of making it work in GitHub-environment.

Reply:
Ok, I'll fix.